rest-api/test/unit_test/offline_test/metrics_export_test.py

```diff
@@ -112,8 +112,9 @@ def _create_data(self):
     with FakeClock(TIME_2):
       # This update to participant has no effect, as the HPO ID didn't change.
       participant = self._participant_with_defaults(
-          participantId=1, version=1, biobankId=2,
+          participantId=1, version=1, biobankId=2, hpoId=3, # <<<< Had to add hpoId here.
```

Comment:
OK, this seems fine.

Reply:
I think the reason this was needed was this line:

https://github.com/all-of-us/raw-data-repository/blob/fb27e8acde5a4a607c657ca7d558b58ca979a4ab/rest-api/test/unit_test/unit_test_util.py#L71

Combined with this line:

https://github.com/all-of-us/raw-data-repository/pull/571/files/fb27e8acde5a4a607c657ca7d558b58ca979a4ab#diff-9bf21946b87489547eac6ffb0cf8e3faR117

I think we could have equally removed the UNSET assignment above (leaving hpoId as None) and it might work fine.

Reply:
If I remember correctly I tried that and a bunch of other tests broke that rely on the UNSET variable